Tech Lead/Minutes/20231024.md

```diff
@@ -0,0 +1,36 @@
+## Technical Lead Call 24 October 2023 (online; special - ESMValCore=2.10 release, part 2)
+
+## Participants
+- Attendees: @ehogan (no voice) @valeriupredoi @bouweandela @zklaus
+- Absentees: @sloosvel @schlunma
+- Chairs and minutes takers: @valeriupredoi
+
+  - Names legend:
+    - Bouwe Andela [@bouweandela](https://github.com/bouweandela)
+    - Klaus Zimmermann [@zklaus](https://github.com/zklaus)
+    - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
+    - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
+    - Manuel Schlund [@schlunma](https://github.com/schlunma)
+    - Emma Hogan [ehogan](https://github.com/ehogan)
+
+## Pending Action Items
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+|2023.07|@ehogan|KGO recording solutions|See [ESMValTool: discussion: #3247](https://github.com/ESMValGroup/ESMValTool/discussions/3247) for the latest news|
+|2023.07|@bouweandela, @remi-kazeroni|Asking the community to check the output once for all| @ehogan: see [ESMValTool: discussion: #3247](https://github.com/ESMValGroup/ESMValTool/discussions/3247) for some additional thoughts on this|
+|2023.07|@valeriupredoi|Shared gmail account to handle to Zenodo credentials|pending|
+|2023.09|@schlunma|Talk about ERA5|pending|
+
+## Topics Discussed
+
+- @zklaus: release situation for 2.10
+  - things are a bit more difficult this time round (new names for obs4MIPS, changes to recipe handling) - @bouweandela fixed a couple issues; further help is needed
+  - Some files are not found in the ``download`` directory if the ESGF node where the data is from is down
+  - @bouweandela (and us) agrees we should get the ESMValCore relase candidate out anyway - we'll test all recipes when CEDA comes bac online
+  - PRs to change obs4MIPS name changes (by Romain) - @bouweandela mentions we should merge them to get the data finding to be using modern names
+  - a couple bottom lines: we'll test everything that don't need CEDA, and we'll get Core RC out
+- @bouweandela: ESMValBot and the VM (former: Remi): we need one maintainer for these: V @valeriupredoi volunteers, Bouwe will meet V at some point talk about it;
+- Mambafore: we'll stick with it for now, until major problems appaear (if any)
```

Comment:
```suggestion
- Mambafore: we'll stick with it for now, until major problems appear (if any)
```

Reply:
haa! You missed one too Mambafore -> Mambaforge :rofl: